src/index.ts

```diff
@@ -72,6 +71,18 @@ export = (app: Probot) => {
     };
     await k8sApi.createNamespacedSecret(namespace,secret);
   });
+
+  // Respond to the GitHub app installation deleted
+  app.on("installation.deleted", async (context) =>{
+
+    // ? user story: delete repo or just the secret ?
```

Comment:
is deleting just the secret enough (following the use case, yes)? Do we need more clean up?
[User unsubscribe](https://github.com/open-services-group/peribolos-as-a-service/blob/fab17cbeb13e9ec6d95ffed4b7952089558583cd/architecture.md#user-unsubscribe)

Reply:
for now it is enough.
acting on an repo and getting the secret can become a problem.
but basic idea would be:
Run a task deleting the peribolos.yaml from .github repo and opening a PR.